plugin-build/src/main/kotlin/io/sentry/android/gradle/util/Versions.kt

```diff
@@ -15,11 +15,13 @@ internal object SentryVersions {
     internal val VERSION_PERFORMANCE = SemVer(4, 0, 0)
     internal val VERSION_OKHTTP = SemVer(5, 0, 0)
     internal val VERSION_FILE_IO = SemVer(5, 5, 0)
+    internal val VERSION_COMPOSE = SemVer(6, 6, 0)
```

Comment:
```suggestion
    internal val VERSION_COMPOSE = SemVer(6, 7, 0)
```
Since the new argument-less method is not available before 6.7.0

Reply:
Exactly! I just reverted it back for now to see if the tests are still failing 😊 

- [ ] Switch to latest version once released